config/default.json

```diff
@@ -1,7 +1,5 @@
 {
-  "smtp": {
-    "host": "127.0.0.1",
-    "port": 25,
-    "sender": "Firefox Accounts <accounts@firefox.com>"
-  }
+  "sender": "Firefox Accounts <accounts@firefox.com>",
+  "smtp_host": "127.0.0.1",
```

Comment:
SAD FACE

Reply:
TBH I think this loss of nesting is my largest concern about this vs #15

Reply:
:(